crates/extensions/tedge_config_manager/src/config.rs

```diff
@@ -39,6 +39,9 @@ pub struct ConfigManagerConfig {
 
     /// If enabled, config file updates are deployed by tedge-write.
     pub use_tedge_write: TedgeWriteStatus,
+
+    pub is_config_snapshot_enabled: bool,
+    pub is_config_update_enabled: bool,
```

Comment:
Something is up-side-down here.

Turning off support for configuration should not be done *inside* the configuration manager but *outside*  by simply not instantiating  the manager.

Reply:
If both `config_snapshot` and `config_update` operations are disabled then yes, we just don't instantiating the config manager. However, the solution above is related to issue when one operation is disabled and one enabled.

Reply:
Okay. However, as it makes little sense to allow `config_update` and not `config_snapshot`, I will keep only one setting to enable writes i.e. configuration updates.